src/main/java/com/dreammaster/gthandler/recipes/AlloySmelterRecipes.java

```diff
@@ -8,236 +11,343 @@
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x3_AluminiumDusts,
+                x1_CopperDust,
+                x4_AluminiumBrassIngots,
+                5 * SECONDS,
+                (int) TierEU.RECIPE_MV);
+
+    }
+
+    public void registerCoreModRecipes() {
+        ItemStack x4_MicaBasedSheets = CustomItemList.MicaBasedSheet.get(4L);
+
+        ItemStack x1_SiliconDioxideDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L);
+
+        ItemStack x4_MicaInsulatorSheets = CustomItemList.MicaInsulatorSheet.get(4L);
+
+        ItemStack x1_RubyDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Ruby, 1L);
+
+        ItemStack x1_SapphireDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sapphire, 1L);
+
+        ItemStack x1_GreenSapphire = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_NetherQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_CertusQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1L);
+
+        ItemStack x2_AluminoSilicateWools = CustomItemList.AluminoSilicateWool.get(2L);
+
+        ItemStack x2_QuartziteDusts = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Quartzite, 2L);
+
+        ItemStack x1_GlassDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glass, 1);
+
+        ItemStack x1_GlassTube = ItemList.Circuit_Parts_Glass_Tube.get(1L);
+
+        ItemStack x2_IridiumReinforcedPlates = GT_ModHandler.getModItem("IC2", "itemPartIridium", 2L);
+
+        ItemStack x3_IridiumAlloyItemCasings = com.dreammaster.item.ItemList.IridiumAlloyItemCasing.getIS()
+                .splitStack(3);
+
+        ItemStack x1_Clay = new ItemStack(Items.clay_ball);
+
+        ItemStack x2_CokeOvenBricks = com.dreammaster.item.ItemList.CokeOvenBrick.getIS().splitStack(2);
+
+        ItemStack x1_ObsidianBlock = new ItemStack(Blocks.obsidian, 1, 0);
+
+        ItemStack x1_ObsidianIngot = GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Obsidian, 2L);
+
+        ItemStack shapeMoldIngot = ItemList.Shape_Mold_Ingot.get(0L);
+        ItemStack shapeMoldBall = ItemList.Shape_Mold_Ball.get(0);
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x4_MicaBasedSheets,
+                x1_SiliconDioxideDust,
+                x4_MicaInsulatorSheets,
+                20 * SECONDS,
+                (int) (TierEU.RECIPE_LV / 2));
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x1_RubyDust,
+                x1_SiliconDioxideDust,
+                x2_AluminoSilicateWools,
+                60 * SECONDS,
+                (int) (TierEU.RECIPE_LV / 2));
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x1_SapphireDust,
+                x1_SiliconDioxideDust,
+                x2_AluminoSilicateWools,
+                60 * SECONDS,
+                (int) (TierEU.RECIPE_LV / 2));
```

Comment:
I think you can make this easier and remove the risk of accidents from integer maths by having a method that takes `tier` and `amps` and returns an appropriate EU/t value. 

E.g. `TierEU.RECIPE_LV.amps(0.5)` and since there are plenty of recipes that use one specific fraction, TierEU.RECIPE_LV.almostOneAmp()` to get 30EU/t and its tiered derivatives.

Reply:
`TierEU.LV` is 32 EU/t, while `TierEU.RECIPE_LV` is 30 EU/t. I'd be in favor to unify them, as the confusion can quickly be made